tests/acceptance/features/apiSpacesShares/shareSpaces.feature

```diff
@@ -239,14 +288,51 @@ Feature: Share spaces
   Scenario Outline: User increases permissions for one member of the group or for the entire group
     Given group "sales" has been created
     And the administrator has added a user "Brian" to the group "sales" using GraphApi
-    And user "Alice" has shared a space "share space" to <firstRecipient> with role "viewer"
+    And user "Alice" has shared a space "share space" with settings:
+      | shareWith | <firstRecipient> |
+      | shareType | <firstShareType> |
+      | role      | viewer           |
     When user "Brian" uploads a file inside space "share space" with content "Test" to "test.txt" using the WebDAV API
     Then the HTTP status code should be "403"
-    When user "Alice" shares a space "share space" to <secondRecipient> with role "editor"
+    When user "Alice" has shared a space "share space" with settings:
+      | shareWith | <secondRecipient> |
+      | shareType | <secondShareType> |
+      | role      | editor            |
     Then the HTTP status code should be "200"
     When user "Brian" uploads a file inside space "share space" with content "Test" to "test.txt" using the WebDAV API
     Then the HTTP status code should be "201"
     Examples:
-      | firstRecipient | secondRecipient |
-      | group "sales"  | user "Brian"    |
-      | user "Brian"   | group "sales"   |
+      | firstRecipient | firstShareType | secondRecipient | secondShareType |
+      | sales          | 8              | Brian           | 7               |
+      | Brian          | 7              | sales           | 8               |
```

Comment:
can we have human readable share type here?

Reply:
yes, I'll split test for 2. It will be more readable